@xen-orchestra/lite/src/composables/fetch-stats.composable.ts

```diff
@@ -35,12 +35,19 @@ export default function useFetchStats<T extends XenApiHost | XenApiVm, S>(
           return;
         }
 
-        const newStats = (await STORES_BY_OBJECT_TYPE[type]().getStats(
+        const objectStore = await STORES_BY_OBJECT_TYPE[type]();
```

Comment:
```suggestion
        const objectStore = STORES_BY_OBJECT_TYPE[type]();
```
`useHostStore` and `useVmStore` are not async function.